README.md

```diff
@@ -172,7 +172,101 @@ when they expire.
 **Important**: ``credentials.state`` changes after a token swap. Be sure to persist
 the new state.
 
-
+### Using OAuth2 Credentials
+
+OAuth2 Credentials work similarly to the 3-step OAuth used by
+Partner Applications. OAuth2 tokens have a 30 minute expiry, 
+but can be swapped for a new token at any time. Procedure for authenticating
+credentials:
+
+ 1) Construct an `OAuth2Credentials` instance. The callback URI should
+    be served by your app in order to complete the authorization:
+    ```python
+    >>> from xero.auth import OAuth2Credentials
+    >>>
+    >>> credentials = OAuth2Credentials(client_id, client_secret,
+    >>>                                 callback_uri=callback_uri, scope=scope)
+    ...
+    ```
+    `scope` should be the list of scopes used by the API. By default this is
+    `['openid', 'profile', 'email']`.
+
+ 2) Get the state and authentication URL, then redirect the user to it:
+
+    ```python
+    >>> from django.http import HttpResponseRedirect
+    >>> from django.core.cache import caches
```

Comment:
While I certainly advocate using Django :-P, this demo should call out the fact that this API isn't Django specific, and give broad instructions that are framework agnostic (i.e., redirect the user to `credentials.generate_url()`, and store `credentials.state` somewhere accessible; for example, in Django, you might use the following...)

Reply:
I've addressed this in the documentation update.